Documentation/ceph-best-practices.md

```diff
@@ -0,0 +1,516 @@
+---
+title: BestPractices
+weight: 3950
+indent: true
+---
+
+# Rook-Ceph Best Practices
```

Comment:
This is a very nice, detailed, doc with a lot of best practices. But it feels like the title should say it's an advanced walkthrough. Best practices that are more common scenarios feels like we need to document in other places such as ceph-cluster-crd.md. 
```suggestion
# Advanced Ceph Walkthrough
```